guides/v2.2/ui_comp_guide/components/ui-columns.md

```diff
@@ -33,3 +33,107 @@ The Columns component is a collection of columns. It renders the `<table>` eleme
 Extends [`uiCollection`]({{ page.baseurl }}/ui_comp_guide/concepts/ui_comp_uicollection_concept.html):
 
 -  [`Magento/Ui/view/base/web/js/grid/listing.js`]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Ui/view/base/web/js/grid/listing.js)
+-  [`Magento/Ui/view/base/web/templates/list/listing.html`]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Ui/view/base/web/templates/list/listing.html)
+
+## Examples
+
+### Integrate Columns component with Listing component
```

Comment:
Hi @serhiyzhovnir. To bring some clarity, I would suggest adjusting the title as following 
```suggestion
### Integrate Columns component as grid (default) with Listing component
```

So, at first glance, the reader understands the core difference between the examples.

Thank you!